docs/tools/kubernetes.md

```diff
@@ -177,40 +105,34 @@ connection to PostgreSQL from Kubernetes.
          name: postgres
          command: ['psql', '$(DATABASE_URI)', '-c', 'SELECT version();']
 
-         # the pg-connection secret becomes environment variables
+         # The pg-connection secret becomes environment variables
          envFrom:
          - secretRef:
              name: pg-connection
    ```
 
-   The connection information in this case, the PostgreSQL service URI, is
-   automatically created by the operator within a Kubernetes secret named
-   after the value from the `connInfoSecretTarget.name` field.
-
-1. Apply the resource to create the pod and test the connection:
+1. Apply the resource to create the pod and test the connection by running:
 
    ```bash
    kubectl apply -f pod-psql.yaml
    ```
 
-1. Check the logs with:
+1. To check the logs, run:
 
    ```bash
    kubectl logs psql-test-connection
    ```
 
-An Aiven for PostgreSQL service deployed through Kubernetes.
-
 ### Clean up
 
-To destroy the resources created:
+1. To destroy the resources, run:
 
 ```bash
 kubectl delete pod psql-test-connection
 kubectl delete postgresqls.aiven.io pg-sample
 ```
```

Comment:
This code block and the one that follows need to be indented for proper alignment as list items.